### PR TITLE
fix(android, events): fix an issue preventing Event.RemoteSeek from b…

### PR DESCRIPTION
**Pull request by jspizziri**
…eing emitted on Android

https://github.com/doublesymmetry/react-native-track-player/issues/1548

### CODE REVIEW

**Comment by jspizziri** [comment]
The build is failing because it relies on the updates to `KotlinAudio` which need to be merged, released, and pulled in here with an updated dependency.

**Review by bradleyflood — DISMISSED** [review]

**Comment by mpivchev** [comment]
KA version needs to be changed to `v0.1.35` for this PR to work:

```
    implementation 'com.github.DoubleSymmetry:KotlinAudio:v0.1.35'
```

**Review by mpivchev — APPROVED** [review]
LGTM!

File: android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt [hunk 1]
```diff
@@ -421,6 +421,12 @@ class MusicService : HeadlessJsTaskService() {
                             emit(MusicEvents.BUTTON_SET_RATING, this)
                         }
                     }
+                    is MediaSessionCallback.SEEK -> {
+                        Bundle().apply {
+                            putDouble("position", it.position.toDouble())
```
Comment (CDBridger) [inline android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt:426]: should this not be it.position.toSeconds()

**Pull request merged**